ggshield/verticals/secret/secret_scan_collection.py

```diff
@@ -70,6 +77,18 @@ def censor(self) -> None:
     def has_policy_breaks(self) -> bool:
         return self.scan.has_policy_breaks
 
+    def policy_breaks(
+        self, *, diff_kinds: Optional[List[DiffKind]] = None
+    ) -> List[PolicyBreak]:
+        policy_breaks = self.scan.policy_breaks
+        if diff_kinds is not None and self.scan.is_diff:
+            policy_breaks = [
+                policy_break
+                for policy_break in self.scan.policy_breaks
+                if policy_break.diff_kind in diff_kinds
+            ]
+        return policy_breaks
```

Comment:
nit: can be slightly simplified to:

```suggestion
        if diff_kinds is not None and self.scan.is_diff:
            return [
                policy_break
                for policy_break in self.scan.policy_breaks
                if policy_break.diff_kind in diff_kinds
            ]
        return self.scan.policy_breaks
```